consensus/src/messages/handlers.rs

```diff
@@ -27,16 +27,24 @@ use crate::sync::live::{
     state_queue::{Chunk, RequestChunk, ResponseChunk},
 };
 
+impl RequestMacroChain {
+    const MAX_LOCATORS: usize = 255;
```

Comment:
For the locators, we are usually putting the election block and one more macro block max. I think this can be significantly smaller.